python/tests/test-3-basis.py

```diff
@@ -122,6 +126,7 @@ def test_304(ceed_resource, capsys):
         print("%12.8f" % lam[i])
 
     stdout, stderr, ref_stdout = check.output(capsys)
+# TODO: fix this for float or double
```

Comment:
refactored

Reply:
These changes have been brought over from main, so I'm marking this as resolved